doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -226,6 +233,89 @@ and ``coqtop``, unless stated otherwise:
   currently associated color and exit.
 :-h, --help: Print a short usage and exit.
 
+
+
+Compiled interfaces (produced using `-vos`)
+----------------------------------------------
+
+
+Compiled interfaces helps saving time while developing Coq formalizations,
+by compiling the formal statements exported by a library independently of
+the proofs that it contains.
+
+   .. warning::
+
+      Compiled interfaces should only be used for development purposes.
+      At the end of the day, one still needs to proof check all files
+      by producing standard `.vo` files. (Technically, when using `-vos`,
+      fewer universe constraints are collected.)
+
+   .. warning::
+
+      In the current implementation, the use of the command `Include` for
+      importing modules compiled using `-vos` might not work properly.
+
+
+**Typical usage.**
+
+Assume a file ``foo.v`` that depends on two files ``bar1.v`` and ``bar2.v``. The
+command ``make foo.requires_vos`` will compile ``bar1.v`` and ``bar2.v`` using
```

Comment:
I'm not sure I understand this sentence. Is this something supported by `coq_makefile`?

Reply:
(`required_vos` would make more sense.)